language-extensions/dotnet-core-CSharp/sample/SubModuleCall/CallSubModule.cs

```diff
@@ -0,0 +1,83 @@
+//*********************************************************************
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+//
+// @File: CallSubModule.cs
+//
+// Purpose:
+//  Sample C# code which can dynamically load a class in SQL Server Language extension. 
+//  This can be used to wrap around different functionalities  to be extended on top of SQL server.
```

Comment:
```suggestion
//  This can be used to wrap around different functionality to be extended on top of SQL server.
```

What does "this" refer to? the custom submodule dll?  This phrasing is awkward/confusing: "This can be used to wrap around different functionality"